diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index 7149c64..bd8519c 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -378,6 +378,7 @@ module.exports = {
         'LogBox is enabled by default so there is no need to call unstable_enableLogBox() anymore. This is a no op and will be removed in the next version.',
       );
   },
+  
   // Deprecated Prop Types
   get ColorPropType(): $FlowFixMe {
     console.error(
